api/vpc_peering.go

```diff
@@ -39,10 +45,22 @@ func (api *API) ReadVpcInfo(instanceID int) (map[string]interface{}, error) {
 	if err != nil {
 		return nil, err
 	}
-	if response.StatusCode != 200 {
-		return nil, fmt.Errorf("ReadInfo failed, status: %v, message: %s", response.StatusCode, failed)
-	}
 
+	statusCode := response.StatusCode
+	log.Printf("[DEBUG] go-api::vpc_peering::info statusCode: %d", statusCode)
+	switch {
+	case statusCode == 400:
+		if strings.Compare(failed["error"].(string), "Timeout talking to backend") == 0 {
```

Comment:
Future improvement here would be our API returning a specific error code for timeouts, so we don't have to trust our API to not ever change the wording of this message. Made a note of it in https://trello.com/c/JnHydMSa.